Orange/projection/pca.py

```diff
@@ -137,16 +153,18 @@ def partial_fit(self, data):
         return self
 
 
-class Projector:
-    def __init__(self, projection, feature):
+class Projector(ComputeValueWCommon):
+    """Transform into a given PCA component."""
+
+    def __init__(self, projection, feature, commonfn):
         self.projection = projection
         self.feature = feature
         self.transformed = None
+        self.commonfn = commonfn #domain conversion to original data set,
+                                 #transform to PCA space
 
-    def __call__(self, data):
-        if data.domain != self.projection.pre_domain:
-            data = data.from_table(self.projection.pre_domain, data)
-        return self.projection.transform(data.X)[:, self.feature]
```

Comment:
I don't currently have a better idea. :confused: I just ~~feel~~know this isn't something a user of the projector should worry about.